### PR TITLE
[Component with its own class name as service type] LPS-200037 Refactor Builder and helper class form asset module

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-200037

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/portlet/AssetPublisherPortlet.java [hunk 1]
```diff
@@ -119,185 +140,6 @@
 )
 public class AssetPublisherPortlet extends MVCPortlet {
 
-	public void getFieldValue(
```
Comment (rafaprax) [inline modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/portlet/AssetPublisherPortlet.java:124]: Hey @MrJohn1911, first make the methods from `AssetPublisherPortlet` private and then, in another commit, move methods from `AssetRSSHelper` to here.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
fix and resend

**Pull request closed without merging**